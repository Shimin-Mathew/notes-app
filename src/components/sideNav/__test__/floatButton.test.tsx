import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { render, cleanup } from '@testing-library/react';
import FloatButton from '../floatButton';

afterEach(cleanup);

test('FloatButton to match snapshot', () => {
	const { asFragment } = render(
		<BrowserRouter>
			<FloatButton label="+" />
		</BrowserRouter>
	);
	expect(asFragment()).toMatchSnapshot();
});
