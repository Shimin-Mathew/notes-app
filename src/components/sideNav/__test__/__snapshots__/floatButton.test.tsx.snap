// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FloatButton to match snapshot 1`] = `
<DocumentFragment>
  <a
    class="btn-float"
    href="/notes/new"
  >
    +
  </a>
</DocumentFragment>
`;
